### PR TITLE
Remove error when attempting to re-initialize TinyliciousClient

### PR DESCRIPTION
**Pull request by sumedhb1995**
Throwing an error was causing issues for React hot reload. Instead of throwing an error, we now just show a log statement instead to let the user know that the earlier instance will be reused

### CODE REVIEW

File: experimental/framework/tinylicious-client/src/TinyliciousClient.ts [hunk 1]
```diff
@@ -123,13 +123,15 @@ let globalTinyliciousClient: TinyliciousClientInstance | undefined;
 export const TinyliciousClient = {
     init(serviceConnectionConfig?: TinyliciousConnectionConfig) {
         if (globalTinyliciousClient) {
-            throw new Error(
-                "TinyliciousClient cannot be initialized more than once",
+            console.log(
+                `TinyliciousClient was attempted to be initialized again. Using existing instance of
```
Comment (micahgodbolt) [inline experimental/framework/tinylicious-client/src/TinyliciousClient.ts:127]:
```suggestion
                `TinyliciousClient has already been initialized. Using existing instance of
```

Comment (sumedhb1995) [reply experimental/framework/tinylicious-client/src/TinyliciousClient.ts]: Updated!

**Comment by micahgodbolt** [comment]
this will need to be updated due to https://github.com/microsoft/FluidFramework/pull/6031

or maybe add this change to that PR

**Comment by sumedhb1995** [comment]
@micahgodbolt Updated this PR with the latest changes from #6031

**Review by micahgodbolt — APPROVED** [review]

**Pull request merged**